items/items.json

```diff
@@ -1,129 +1,109 @@
 [
-	// Characters
 	{
 		"name": "Crono",
 		"type": "toggle",
-		"loop": "true",
```

Comment:
Was this tested in EmoTracker, or just Poptracker?  It's probably fine, but I want to make sure it doesn't cause any issues with EmoTracker since I think that's what most users are running.

Reply:
PopTracker (I haven't had success getting EmoTracker working with wine on my setup). I can revert it that for now, to minimize risk.

It was because it failed strict JSON schema validation. I cannot tell if PopTracker is 100% schema-compatible with EmoTracker (although I gather that's intent). It looks like "loop" is supposed to be a boolean and only for certain elements, not toggles.

Reply:
I can test it out on my end and let you know.  Leave it as is for now.  It's entirely possible that element isn't valid for toggles in EmoTracker either and is just being ignored.

Reply:
Cool. Thanks.

Reply:
It looks like it does work for toggles on EmoTracker.  When `true`, it allows left click to toggle the item state back off.  When `false`, you have to right click to unset the toggle.  Honestly, I'm fine either way.  Most people use the autotracker, and I imagine manually unchecking a toggle is pretty rare.

Reply:
Ah. Thanks for checking. Reverted that change, set to non-strict on items.json (like layouts, locations).